### PR TITLE
reduce hero padding and card width

### PR DESCRIPTION
**Pull request by texas-kristen**
Fix for #1515 

My local environment always makes the site look smaller, so I can't tell if the 2/3rds width looks *too* small, going to let the deploy preview run and see how it looks.

### CODE REVIEW

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit 5422ee20c32761650321585aa8e3705d47df3892

https://deploy-preview-1516--stoplightio.netlify.app

**Comment by texas-kristen** [comment]
@paulatulis I'm trying to fix #1515 , and on my local environment it looks great, but the deploy preview looks identical to the behavior on the site now - what am I missing here? 

I didn't want to accidentally affect other areas, so I  kept the changes contained to the Pricing Plans component and the Pricing template.

**Comment by paulatulis** [comment]
I think you changed the `border-b` width instead (which also looks fine). The issue is with the width of the cards. I changed [this line](https://github.com/stoplightio/stoplight.io/pull/1516/files#diff-ed072f0302c59eef68595f995816b0daR68) to have to `w-64 md:w-96` instead of `w-80` and it's not extending off the sides when I make my screen a little smaller, but it is still responsive with the `md` and `sm` screen sizes. Want to give that a shot and let me know if that makes it better?

**Comment by texas-kristen** [comment]
@paulatulis at least in the deploy preview, both issues are still happening for me 
image

**Comment by paulatulis** [comment]
Apologies I meant to tell you to try it and see if it worked. I didn't push it up. I just made the change AND pushed it up so now hopefully it renders. Let me know!

File: src/templates/Pricing/index.tsx [hunk 1]
```diff
@@ -31,13 +31,13 @@ export const Pricing: React.FunctionComponent<IPricing> = ({
 }) => {
   return (
     <Layout>
-      {hero && <LandingHero bgColor={color} {...hero} containerClassName="pb-64" />}
+      {hero && <LandingHero bgColor={color} {...hero} containerClassName="pb-40" />}
       {plans && <PricingPlans color={color} plans={plans} />}
-      <div className="container w-3/4 border-b"></div>
+      <div className="container w-3/5 border-b"></div>
```
Comment (paulatulis) [inline src/templates/Pricing/index.tsx:36]: We should make these consistent, so I'd change this to w-2/3

**Pull request merged**